dnsdbq.c

```diff
@@ -334,27 +350,30 @@ static bool sort_bydata = false;
 static writer_t writers = NULL;
 static int exit_code = 0; /* hopeful */
 static size_t ideal_buffer;
+static bool iso = false;
```

Comment:
I assume this is not a switch to use OSI networking, so maybe clarify the variable name or comment it?